Resources/Prototypes/radio_channels.yml

```diff
@@ -95,3 +95,17 @@
   color: "#f6ce64"
   # long range since otherwise it'd defeat the point of a handheld radio independent of telecomms
   longRange: true
+
+- type: radioChannel
+  id: Bugged1
+  name: chat-radio-bugged1
+  keycode: '0' #its never talked on,so this is kinda pointless, but the game shits if I don't specify one.
```

Comment:
I tried this, unfortunately without a pretty annoying refactor of chat system I don't think I can get rid of it since it repeatedly checks for a radio channels to have a keycode saved in a dictionary for things like examining headsets and whatnot. Also, annoyingly, I don't think I can set it to a normal dummy key like F24 or anything either since it just pulls the first character.